routes.yaml

```diff
@@ -0,0 +1,6 @@
+app_name: lfp-appeals-frontend
+group: web
+weight: 900
+routes:
+  1: /appeal-a-penalty 
```

Comment:
Where did you take it from?

Reply:
5th Feb conversation on the lfp-appeals slack channel